internal/provider/private_endpoint_services_resource_test.go

```diff
@@ -36,7 +36,7 @@ import (
 func TestAccDedicatedPrivateEndpointServicesResource(t *testing.T) {
 	t.Parallel()
 	clusterName := fmt.Sprintf("%s-endpt-svc-%s", tfTestPrefix, GenerateRandomString(2))
-	testPrivateEndpointServicesResource(t, clusterName, false /* useMock */, false /* isServerless */)
+	testPrivateEndpointServicesResource(t, clusterName, false /* useMock */, false /* isServerless */, client.CLOUDPROVIDERTYPE_GCP)
 }
```

Comment:
should we also run a dedicated test for AWS?

Reply:
Serverless is already covering the logic for acceptance tests here since it's the same for Serverless and Dedicated.